modules/Microsoft.Sql/managedInstances/.test/common/dependencies.bicep

```diff
@@ -0,0 +1,350 @@
+@description('Required. The name of the Virtual Network to create.')
+param virtualNetworkName string
+
+@description('Required. The name of the Network Security Group to create.')
+param networkSecurityGroupName string
+
+@description('Required. The name of the Route Table to create.')
+param routeTableName string
+
+@description('Required. The name of the Managed Identity to create.')
+param managedIdentityName string
+
+@description('Required. The name of the Key Vault to create.')
+param keyVaultName string
+
+@description('Optional. The location to deploy resources to.')
+param location string = resourceGroup().location
+
+var sqlMiVnetAddressPrefix = '10.0.0.0/16'
```

Comment:
why the variable(s) if they're only used in one single location?

Reply:
Same as other comment